src/components/SubmitButton.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import "../styles/Button.css";
+
+function CustomButton({ uploadOnClick }) {
```

Comment:
will the functionality of this button change according to the function that is passed as a parameter?